aws/resource_aws_cloudwatch_event_target_test.go

```diff
@@ -648,6 +686,100 @@ CONTAINER_PROPERTIES
 `, rName)
 }
 
+func testAccAWSCloudWatchEventTargetConfigKinesis(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_cloudwatch_event_rule" "cloudwatch_event_rule" {
+  name        = "%[1]s"
+  description = "schedule_batch_test"
+  schedule_expression = "rate(5 minutes)"
+}
+
+resource "aws_cloudwatch_event_target" "test" {
+  arn = "${aws_kinesis_stream.kinesis_stream.arn}"
+  rule = "${aws_cloudwatch_event_rule.cloudwatch_event_rule.id}"
+  role_arn = "${aws_iam_role.iam_role.arn}"
+
+  kinesis_target {
+    partition_key_path = "$.detail"
+  }
+
+  depends_on = [
+    "aws_kinesis_stream.kinesis_stream",
+    "aws_iam_role.iam_role",
+  ]
+}
+
+resource "aws_iam_role" "iam_role" {
+  name = "event_%[1]s"
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "events.amazonaws.com"
+      }
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_kinesis_stream" "kinesis_stream" {
+  name = "%[1]s"
+}
+`, rName)
+}
+
+func testAccAWSCloudWatchEventTargetConfigSqs(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_cloudwatch_event_rule" "cloudwatch_event_rule" {
+  name        = "%[1]s"
+  description = "schedule_batch_test"
+  schedule_expression = "rate(5 minutes)"
+}
+
+resource "aws_cloudwatch_event_target" "test" {
+  arn = "${aws_sqs_queue.sqs_queue.arn}"
+  rule = "${aws_cloudwatch_event_rule.cloudwatch_event_rule.id}"
+  role_arn = "${aws_iam_role.iam_role.arn}"
+
+  sqs_target {
+    message_group_id = "event_group"
+  }
+
+  depends_on = [
```

Comment:
The `depends_on` here is unnecessary since we already have explicit dependencies above 👍